adminer/include/functions.inc.php

```diff
@@ -1437,7 +1439,7 @@ function edit_form($TABLE, $fields, $row, $update) {
 				)
 				: (!$update && $field["auto_increment"]
 					? ""
-					: (isset($_GET["select"]) ? false : $default)
+					: (isset($_GET["select"]) ? false : (empty($field["null"]) ? $default : null))
```

Comment:
What's this?

Reply:
This should be resolved when displaying the default value in the form, not here.

Reply:
fix in new commits.